lib/msf/ui/console/command_dispatcher/creds.rb

```diff
@@ -414,11 +420,13 @@ def creds_search(*args)
              when 'password'
                Metasploit::Credential::Password
              when 'hash'
-               Metasploit::Credential::PasswordHash
+               Metasploit::Credential::NonreplayableHash
```

Comment:
Apparently `hash` corresponds to a Nonreplayable hash instead of a PasswordHash. This change were required otherwise the specs were failing.